mesonbuild/scripts/meson_install.py

```diff
@@ -97,6 +97,12 @@ def restore_selinux_contexts():
         # is ignored quietly.
         return
 
+    try:
+        subprocess.check_call(['restorecon'])
```

Comment:
Why not implement it like this:

```python
if not shutil.which('restorecon'):
    # If we don't have restorecon, failure is ignored quietly.
    return
```

Reply:
Ok I will update my PR with your proposal, thanks for your review.